src/base64.h

```diff
@@ -4,6 +4,7 @@
 #if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
 
 #include "util.h"
+#include "base64-inl.h"
```

Comment:
Can you remove this line, and replace `#include "base64.h"` with `#include "base64-inl.h"` instead where necessary?

The general idea here is:

- `.h` header files contain declarations, and sometimes some definitions that don’t require including other headers (e.g. for getters/setters)
  - `.h` header files include only other `.h` headers
- `-inl.h` header files contain definitions of inline functions from the corresponding `.h` header file, i.e. functions marked `inline` in the declaration or `template` functions
  - `-inl.h` header files include the corresponding `.h` header file, and other `.h` and `-inl.h` header files as needed
  - It’s not always necessary to split out the definitions from the `.h` file into an `-inl.h` file, but it becomes necessary when the contents of other `-inl.h` files start being used, and it is very much recommended when the functions are longer than a few lines to keep the corresponding `.h` file readable
  - All visible definitions from this file should have been declared in the corresponding `.h` header file
- `.cc` files contain definitions of non-inline functions from the corresponding `.h` header file
  - `.cc` files also include the corresponding `.h` header file, and other `.h` and `-inl.h` header files as needed

I hope this is helpful?